test/e2e/framework/k8s.go

```diff
@@ -27,6 +27,18 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+// Updates ingress-nginx deployment to add `--enable-dynamic-configuration` to the args
+// and wait for it to restart
+func (f *Framework) EnableDynamicConfiguration() error {
+	return nil
+}
+
+// Updates ingress-nginx deployment to remove `--enable-dynamic-configuration` from the args
+// and wait for it to restart
+func (f *Framework) DisableDynamicConfiguration() error {
+	return nil
+}
```

Comment:
@aledbf implementation of both of these functions requires us to restart ingress-nginx during test run. This will introduce some latency. I'm tempted to change enabling live configuration update from command line argument to global configmap. That way we would need to only reload Nginx and avoid restart of ingress-nginx. What do you think? If you are OK with this, in a separate PR I'll delete `--enable-dynamic-configuration` command line flag and make it configurable through the configmap instead.

Reply:
@ElvinEfendi let's keep this behind a flag for now. Don't worry about the latency in the tests.